src/Components/Samples/BlazorUnitedApp/Program.cs

```diff
@@ -22,11 +23,12 @@
 }
 
 app.UseHttpsRedirection();
-
+app.UseBlazorFrameworkFiles();
 app.UseStaticFiles();
 
 app.UseRouting();
 
 app.MapRazorComponents<App>();
+app.MapBlazorHub();
```

Comment:
Could we avoid modifying the sample? This is adding work that we'll have to undo as we plug in additional stuff.

Reply:
Definitely, my [plan](https://github.com/dotnet/aspnetcore/pull/48294#discussion_r1197202826) was to revert changes to the sample. The changes were just to make testing convenient.